lib/blockscore/errors/invalid_request_error.rb

```diff
@@ -15,15 +15,17 @@ class InvalidRequestError < APIError
     # end
     def initialize(response)
       super
-      @param = @http_body[:error][:param]
+      @param = error_advice[:param]
```

Comment:
Instance of https://github.com/BlockScore/blockscore-ruby/pull/56/files#r52110298